### DISCUSSION TITLE
Time series with multi-dimensional features

### DISCUSSION THREAD

**Discussion by kiskani** [oliverguhr/transformer-time-series-prediction#8] [General]
Thanks for the great repository and code! The proof of concept is for scalar feature inputs like temperature data. Are there any efforts to extend this into multi-dimensional feature vectors as input data?

**Comment by oliverguhr** [comment]
I am working on this right now. Do you have any suggestion for a standard data set to test this?

**Reply by kiskani** [reply]
Yes, as a matter of fact I do. Please try it on this dataset:
https://www.kaggle.com/c/jane-street-market-prediction/data

**Reply by oliverguhr** [reply]
Great idea, thank you for that link. But I'll guess I should keep the solution to me if the model works well :)

**Reply by kiskani** [reply]
Yeah, don't worry! I am working on my own solution. Was just curious to see if your solution is extendable to multi-dimensional inputs.
